snapcast fails to compile with boost 1.81.0

**Describe the bug**
snapcast fails to compile with boost 1.81.0
- core::string_view no longer supports the .to_string() or .clear() extensions from Utility

https://github.com/boostorg/beast/issues/2417
https://github.com/boostorg/beast/pull/2418
https://github.com/boostorg/beast/commit/9d23bec2bc523223a377582302bd28d04cb514ed
https://github.com/boostorg/beast/issues/2594


**Steps to Reproduce**
1. Install boost 1.81.0

**Environment details**
- OS: LibreELEC 11
- Snapcast version: 0.26.0
- Installed from a package, self compiled, ...
  - self compiled 

**Attach logfile if applicable**
Generate logs with `snapclient --logfilter debug` or `snapserver --logging.filter debug` if possible and paste them in the following codeblock

```log

/var/media/DATA/home-rudi/LibreELEC.kernel11/build.LibreELEC-ARMv7.arm-11.0-devel/toolchain/bin/armv7a-libreelec-linux-gnueabihf-g++ -march=armv7-a -mtune=cortex-a8 -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=neon-vfpv3 -Wall -pipe  -O2 -fomit-frame-pointer -DNDEBUG -pthread                       -I/var/media/DATA/home-rudi/LibreELEC.kernel11/build.LibreELEC-ARMv7.arm-11.0-devel/install_pkg/aixlog-1.5.0/usr/include                       -I/var/media/DATA/home-rudi/LibreELEC.kernel11/build.LibreELEC-ARMv7.arm-11.0-devel/install_pkg/asio-1.21.0/usr/include                       -I/var/media/DATA/home-rudi/LibreELEC.kernel11/build.LibreELEC-ARMv7.arm-11.0-devel/install_pkg/popl-1.3.0/usr/include -O2  -std=c++17 -Wall -Wextra -Wpedantic -Wno-unused-function -DBOOST_ERROR_CODE_HEADER_ONLY -DHAS_FLAC -DHAS_OGG -DHAS_VORBIS -DHAS_VORBIS_ENC -DHAS_OPUS -DHAS_SOXR -DVERSION=\"0.26.0\" -I. -I.. -I../common -DHAS_AVAHI -DHAS_DAEMON -DHAS_ALSA -pthread -c control_session_http.cpp -o control_session_http.o
control_session_http.cpp: In function 'std::string {anonymous}::path_cat(boost::beast::string_view, boost::beast::string_view)':
control_session_http.cpp:130:21: error: 'using string_view = boost::core::string_view' {aka 'class boost::core::basic_string_view<char>'} has no member named 'to_string'
  130 |         return path.to_string();
      |                     ^~~~~~~~~
control_session_http.cpp:131:31: error: 'using string_view = boost::core::string_view' {aka 'class boost::core::basic_string_view<char>'} has no member named 'to_string'
  131 |     std::string result = base.to_string();
      |                               ^~~~~~~~~
control_session_http.cpp: In lambda function:
control_session_http.cpp:174:26: error: 'using string_view = boost::core::string_view' {aka 'class boost::core::basic_string_view<char>'} has no member named 'to_string'
  174 |         res.body() = why.to_string();
      |                          ^~~~~~~~~
control_session_http.cpp: In lambda function:
control_session_http.cpp:185:48: error: 'using string_view = boost::core::string_view' {aka 'class boost::core::basic_string_view<char>'} has no member named 'to_string'
  185 |         res.body() = "The resource '" + target.to_string() + "' was not found.";
      |                                                ^~~~~~~~~
control_session_http.cpp: In lambda function:
control_session_http.cpp:207:52: error: 'using string_view = boost::core::string_view' {aka 'class boost::core::basic_string_view<char>'} has no member named 'to_string'
  207 |         res.body() = "An error occurred: '" + what.to_string() + "'";
      |                                                    ^~~~~~~~~
make[1]: *** [Makefile:109: control_session_http.o] Error 1
make[1]: Leaving directory '/var/media/DATA/home-rudi/LibreELEC.kernel11/build.LibreELEC-ARMv7.arm-11.0-devel/build/snapcast-0.26.0/server'
make: *** [Makefile:11: server] Error 2
```

--- a/server/control_session_http.cpp
+++ b/server/control_session_http.cpp
@@ -127,8 +127,8 @@ boost::beast::string_view mime_type(boost::beast::string_view path)
 std::string path_cat(boost::beast::string_view base, boost::beast::string_view path)
 {
     if (base.empty())
-        return path.to_string();
-    std::string result = base.to_string();
+        return static_cast<std::string>(path);
+    std::string result = static_cast<std::string>(base);
     char constexpr path_separator = '/';
     if (result.back() == path_separator)
         result.resize(result.size() - 1);
@@ -171,7 +171,7 @@ void ControlSessionHttp::handle_request(http::request<Body, http::basic_fields<A
         res.set(http::field::server, HTTP_SERVER_NAME);
         res.set(http::field::content_type, "text/html");
         res.keep_alive(req.keep_alive());
-        res.body() = why.to_string();
+        res.body() = static_cast<std::string>(why);
         res.prepare_payload();
         return res;
     };
@@ -182,7 +182,7 @@ void ControlSessionHttp::handle_request(http::request<Body, http::basic_fields<A
         res.set(http::field::server, HTTP_SERVER_NAME);
         res.set(http::field::content_type, "text/html");
         res.keep_alive(req.keep_alive());
-        res.body() = "The resource '" + target.to_string() + "' was not found.";
+        res.body() = "The resource '" + static_cast<std::string>(target) + "' was not found.";
         res.prepare_payload();
         return res;
     };
@@ -204,7 +204,7 @@ void ControlSessionHttp::handle_request(http::request<Body, http::basic_fields<A
         res.set(http::field::server, HTTP_SERVER_NAME);
         res.set(http::field::content_type, "text/html");
         res.keep_alive(req.keep_alive());
-        res.body() = "An error occurred: '" + what.to_string() + "'";
+        res.body() = "An error occurred: '" + static_cast<std::string>(what) + "'";
         res.prepare_payload();
         return res;
     };

